packages/atomic/src/components/commerce/atomic-commerce-layout/atomic-commerce-layout.ts

```diff
@@ -11,6 +11,10 @@ import {buildCommerceLayout} from './commerce-layout';
  * The `atomic-commerce-layout` helps organize elements in the commerce page.
  *
  * @slot default - The default slot where you can add child components to the layout.
+ *
+ * @cssprop --atomic-layout-max-search-box-input-width: The maximum width that the search box input will take.
+ * @cssprop --atomic-layout-max-search-box-double-suggestions-width: The maximum width that the search box suggestions will take when displaying a double list.
+ * @cssprop --atomic-layout-search-box-left-suggestions-width: When displaying a double list, the width of the left list.
```

Comment:
```suggestion
 * @cssprop --atomic-layout-search-box-left-suggestions-width: The width of the left list when displaying a double list.
```

Just for the sake of consistency.